### PR TITLE
Markdown and January news

### PR DESCRIPTION
**Pull request by dantleech**
@dbu and I briefly discussed closing down the PHPCR news and shifting the monthly news here. It makes more sense because CMF covers PHPCR and well, there is simply more news for the CMF. Also we have a twitter handle.

In addition to some news for January, this PR adds support for markdown and loading fixtures from individual files (like a static stite generator).

### CODE REVIEW

File: src/Cmf/MainBundle/Resources/data/posts/2015-02-01.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+format: markdown
+template: CmfMainBundle:Cms:news_detail.html.twig
+publish_start_date: "2015-02-01"
+body: |
+    Its quite an exciting time for the CMF, there is talk about the next
+    generation of Jackalope, the Resource stack, rewriting the
+    TreeBrowserBundle and true multi-site support.
+
+    We also had a CMF hackday in December at the Mayflower offices in Würzburg, Germany and we are
+    hoping to have another one in the coming months, if you are interested make your self known
```
Comment (WouterJ) [inline src/Cmf/MainBundle/Resources/data/posts/2015-02-01.yml]: What do you think about splitting this in more than one sentence? "one in the coming months. If you are interested, make yourself known [...]"

File: src/Cmf/MainBundle/Resources/data/posts/2015-02-01.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    on the [CMF Mailinglist](https://groups.google.com/forum/#!forum/symfony-cmf-devs).
+
+    <center><img src="https://pbs.twimg.com/media/B4rF-8aIcAAQkSB.jpg" width="600" /><br/>
+    <i>CMF hackday</i></center>
```
Comment (WouterJ) [inline src/Cmf/MainBundle/Resources/data/posts/2015-02-01.yml]: ah, we need some proper styling for this with a HTML like:

````html
<figure>
    <img src="https://pbs.twimg.com/media/B4rF-8aIcAAQkSB.jpg" />

    <figcaption>CMF hackday</figcaption>
</figure>
````

File: src/Cmf/MainBundle/Resources/data/posts/2015-02-01.yml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+    ### Resources
+    
+    Resources are a new and experimental concept in the CMF. Resources are a
+    common interface to disparate entities within the system. For example a
+    resource could represent a file on the local filesystem, a Doctrine Entity
+    or a PHPCR Node.
+
+    The resource stack is built upon
+    [Puli](https://github.com/puli/repository).
```
Comment (WouterJ) [inline src/Cmf/MainBundle/Resources/data/posts/2015-02-01.yml]: not sure about the line wrapping thing done here.

File: src/Cmf/MainBundle/Resources/data/posts/2015-02-01.yml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+    Two of the greatest apparent benefits will be nice ways to support multiple
+    sites and dynamic themeing.
```
Comment (WouterJ) [inline src/Cmf/MainBundle/Resources/data/posts/2015-02-01.yml]: typo: themeing

**Comment by dantleech** [comment]
Updated.

**Comment by dantleech** [comment]
ping @dbu @lsmith77 what do you think?

**Comment by lsmith77** [comment]
so from now on, new news items will be in separate files that can contain html or markdown as the content? looks fine to me.

**Comment by lsmith77** [comment]
its deployed http://cmf.symfony.com/news/cmf-news-january-2015

**Comment by dantleech** [comment]
Cool, maybe ping @dbu to announce it on the SymfonyCMF twitter handle

**Comment by dbu** [comment]
that happened. i think we should call start to call those news with the name of the current month, even though they obviously are about the last month. it just sounds like we are late :-)

**Comment by dantleech** [comment]
yeah, I guess it makes more sense that way, lets do that.

**Pull request merged**